integrations/octopus/client.py

```diff
@@ -0,0 +1,102 @@
+from typing import Any, AsyncGenerator, Optional
+from loguru import logger
+from port_ocean.utils import http_async_client
+from httpx import HTTPStatusError, Timeout
+
+PAGE_SIZE = 50  # Number of items to fetch per page
```

Comment:
remove the comment, PAGE_SIZE is self explanatory

Reply:
Done